addons/udes_get_info/models/models.py

```diff
@@ -1,46 +1,59 @@
-# -*- coding: utf-8 -*-
-from odoo import models, _
+from odoo import fields, models, _
 from odoo.addons.udes_common.models import add_if_not_exists
 import logging
 
 _logger = logging.getLogger(__name__)
 
 
-# N.B: Basic values should not have any relations as this is to stop the recursion
-BASIC_GET_INFO_VALUES = {"id", "name", "display_name"}
+# N.B: Basic values that some models have, no need to add them in _get_info_field_names
+BASIC_GET_INFO_VALUES = {"id", "name", "display_name", "code", "barcode"}
```

Comment:
Code and barcode are not in all models, why we add them here?

Reply:
Barcode in some model which we use a lot are(location, product), code generally may be used like a identifier in models in odoo, Advantage: will be no need to add in model fields , Disadvantage: Will log in server for the models that is not.

What do you you think is better, happy to remove and to add barcode in location only at the moment

Reply:
I prefer to avoid warnings and we add it to the specific models.